docs/manual/developer_guide.adoc

```diff
@@ -1337,13 +1337,321 @@ Available low-level Jinja macros that can be used in Bash remediations:
+* Check if Audit is configured to record events that modify account changes.
+* Parameters:
+** *path* - path that should be part of the audit rule as a value of `-w` argument, eg. `/etc/group`.
+* Languages: Ansible, Bash, OVAL
+
+file_groupowner::
+* Check group that owns the given file.
+* Parameters:
+** *filepath* - File path to be checked. If the file path ends with `/` it describes a directory.
+** *file_regex* - Regular expression that matches file names in a directory specified by *filepath*. Can be set only if *filepath* parameter specifies a directory.
+** *filegid* - group ID (GID)
+* Languages: Ansible, Bash, OVAL
+
+file_owner::
+* Check user that owns the given file.
+* Parameters:
+** *filepath* - File path to be checked. If the file path ends with `/` it describes a directory.
+** *file_regex* - Regular expression that matches file names in a directory specified by *filepath*. Can be set only if *filepath* parameter specifies a directory.
+** *fileuid* - user ID (UID)
+* Languages: Ansible, Bash, OVAL
+
+file_permissions::
+* Checks permissions (mode) on a given file.
+* Parameters:
+** *filepath* - File path to be checked. If the file path ends with `/` it describes a directory.
+** *file_regex* - Regular expression that matches file names in a directory specified by *filepath*. Can be set only if *filepath* parameter specifies a directory.
+** *filemode* - File permissions in a hexadecimal format, eg. `'0640'`.
+* Languages: Ansible, Bash, OVAL
+
+grub2_bootloader_argument::
+* Checks kernel command line arguments in GRUB 2 configuration.
+* Parameters:
+** *arg_name* - argument name, eg. `audit`
+** *arg_value* - argument value, eg. `'1'`
+* Languages: Ansible, Bash, OVAL
+
+kernel_module_disabled::
+* Checks if the given Linux kernel module is disabled.
+* Parameters:
+** *kernmodule* - name of the Linux kernel module, eg. `cramfs`
+* Languages: Ansible, Bash, OVAL
+
+mount::
+* Checks that a given mount point is located on a separate partition.
+* Parameters:
+** *mountpoint* - path to the mount point, eg. `/var/tmp`
+* Languages: Anaconda, OVAL
+
+mount_option::
+* Checks if a given partition is mounted with a specific option such as "nosuid".
+* Parameters:
+** *mountpoint* - mount point on the filesystem eg. `/dev/shm`
+** *mountoption* - mount option, eg. `nosuid`
+** *filesystem* - filesystem in `/etc/fstab`, eg. `tmpfs`. Used only in Bash remediation.
+** *type* - filesystem type. Used only in Bash remediation.
+** *mount_has_to_exist* - Used only in Bash remediation. Specifies if the *mountpoint* entry has to exist in `/etc/fstab`. If set to `yes` and the *mountpoint* entry is not present in `/etc/fstab` the Bash remediation terminates.
+* Languages: Anaconda, Ansible, Bash, OVAL
+
+mount_option_remote_filesystems::
+* Checks if all remote filesystems (NFS mounts in `/etc/fstab`) are mounted with a specific option.
+* Parameters:
+** *mountpoint* - always set to `remote_filesystems`
```

Comment:
What is the purpose of this parameter if it is ALWAYS set to something?

Reply:
I don't know. I think we should take a closer look at the templates. I have opened an issue in upstream https://github.com/ComplianceAsCode/content/issues/4914